src/index.html

```diff
@@ -11,7 +11,124 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+           required>
```

Comment:
Look through [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) once more, you tags doesn't look right, it should be like:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```